code/modules/projectiles/projectile/rocket.dm

```diff
@@ -45,9 +45,6 @@
 /obj/item/projectile/rocket/lowyield
 	name = "low yield rocket"
 	icon_state = "rpground"
-	damage = 45
```

Comment:
why are you removing these vars from only this subtype when both parent and other children have the var?

Reply:
Reinstated, sorry about that